pkg/features/kube_features.go

```diff
@@ -851,21 +787,13 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	CPUCFSQuotaPeriod: {Default: false, PreRelease: featuregate.Alpha},
 
-	CPUManager: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.26
-
```

Comment:
@cyclinder please remove the other bits pertaining to CPUManager above (around lines 70-75)

Reply:
Oh thanks, I miss it

Reply:
/unhold